mobile_testkit_tests/test_utils.py

```diff
@@ -0,0 +1,10 @@
+from keywords import utils
+import subprocess
+
+
+def test_add_cbs_to_sg_config_server_field():
+    subprocess.call("echo '{\"couchbase_servers\": [\n{\n\"ip\": \"192.168.33.10\",\n \"name\": \"cb1\"\n},\n {\n\"ip\": \"192.168.33.11\",\n\"name\": \"cb2\"\n},\n{\n\"ip\": \"192.168.33.12\",\n\"name\": \"cb3\"\n}\n]}' > resources/cluster_configs/test.json", shell=True)
```

Comment:
Add this as a mock data under mobile_testkit_tests/test_data and use it. We have a lot of mock data files here already.

Reply:
Also use the existing mock data for 1 server too to make sure there is no stale ","